frontend/src/pages/article/edit-article-page.tsx

```diff
@@ -11,6 +11,9 @@ import styles from './styles.module.scss';
 
 const EditArticlePage: React.FC = () => {
   const location = useLocation();
+  if (!location.state) {
+    return <Navigate to={AppRoute.ARTICLES_MY_ARTICLES} />;
+  }
```

Comment:
Are you sure that we need this check as we're checking if article from location state existing right below it